js/src/csp/components/CSPBuilder.vue

```diff
@@ -57,7 +57,7 @@ import { Prop, Watch } from "vue-property-decorator";
 import * as shortid from "shortid";
 
 import CSPToolbar from "./CSPBuilderToolbar.vue";
-import EllipseGraphNode from "../../components/EllipseGraphNode.vue";
+import EllipseGraphNode from "../../components/RoundedRectangleGraphNode.vue";
```

Comment:
Does RoundedRectangleGraphNode.vue still export an EllipseGraphNode?

Reply:
Changed the import url back to Ellipse